scheduler/src/test/resources/application.properties

```diff
@@ -24,6 +24,8 @@ server.ssl.key-store-password=123456
 server.ssl.key-store-type=PKCS12
 server.ssl.trust-store=src/test/resources/certs/test.truststore
 server.ssl.trust-store-password=123456
+server.ssl.enabled-protocols[3]=TLSv1,TLSv1.1,TLSv1.2
```

Comment:
how about other microservices?

Reply:
@boyang9527 security team only report the issue in scheduler and in the other components there is no issue.